pages/11ty/markdown.shortcut.js

```diff
@@ -75,5 +79,6 @@ class MDRenderer {
 
 module.exports = (config) => {
   config.addNunjucksAsyncShortcode('mdRender', MDRenderer.render);
+  config.addNunjucksShortcode('mdWrap', MDRenderer.renderContentString);
```

Comment:
Do we really need it? It's just one class that we can place anywhere, for example, on the item block or even once on the whole component.